15/umbraco-cms/fundamentals/design/templates/README.md

```diff
@@ -150,9 +192,3 @@ The created partial view can now be injected into any template by using the `@Ht
 ### Tutorials
```

Comment:
The Names sections link above takes you to the Github page. Might help updating the link